distribution/springboot_setup/README.md

```diff
@@ -16,20 +16,35 @@ docker-compose up
 ```
 
 ### 2. Import database dumps
-Restore a database archive for each database:
+Restore a database archive for each database by running the approriate command depending on your operating system:
 
 
 - a dump with not much data for the central database
+
+<sub>Linux</sub>
 ```
 cd distribution/springboot_setup/db
 zcat dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
 ```
+<sub>macOS</sub>
+```
+cd distribution/springboot_setup/db
+unzip -p dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
+```
 
 - a dump with lots of data for the remote database.
+
+<sub>Linux</sub>
 ```
 cd distribution/springboot_setup/db
 zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
 ```
+<sub>macOS</sub>
+```
+cd distribution/springboot_setup/db
+unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
+```
+
```

Comment:
@Ruhanga please replace the entire piece of Markdown that I provided under the line [here](https://github.com/openmrs/openmrs-eip/pull/8#discussion_r462112217).

Reply:
Ohh, ok @mks-d. Could you share the raw markdown? I seemingly cannot extract it from your comment above, thanks.

Reply:
@Ruhanga here:
`````
### 2. Import database dumps
Restore a database archive for each database:

##### a dump with not much data for the central database
<sub>Linux</sub>
```bash
cd distribution/springboot_setup/db
zcat dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```bash
cd distribution/springboot_setup/db
unzip -p dump_receiver.zip | docker exec -i db_db_central_1 /usr/bin/mysql -u root --password=root openmrs
```

##### a dump with lots of data for the remote database.
<sub>Linux</sub>
```bash
cd distribution/springboot_setup/db
zcat dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
<sub>macOS</sub>
```bash
cd distribution/springboot_setup/db
unzip -p dump_sender_2.3.zip | docker exec -i db_db_remote_1 /usr/bin/mysql -u root --password=root openmrs
```
`````

Reply:
Thanks @mks-d. I've used the markup.